README.md

```diff
@@ -37,21 +46,25 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v4
-
-    - uses: tidalmigrations/tidal-code-action@v0
-      with:
-        tidal-email: ${{ secrets.TIDAL_EMAIL }}
-        tidal-password: ${{ secrets.TIDAL_PASSWORD }}
-        tidal-url: https://workspace.tidal.cloud
-        app-id: 1
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - uses: tidalmigrations/tidal-code-action@v2
```

Comment:
We skipped v1?

Reply:
Nah, just our docs were out of date.
https://github.com/tidalmigrations/tidal-code-action/releases/tag/v1 😅